cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -164,15 +178,19 @@ cd TFW_testing_single_configure_prototype
 
 if [ "icc" == ${CC:?} ]
 then
-  CONFIG_SCRIPT=PullRequestLinuxIntelTestingSettings.cmake
+    CONFIG_SCRIPT=PullRequestLinuxIntelTestingSettings.cmake
 else
-  if [ "Trilinos_pullrequest_gcc_4.8.4" == "${JOB_BASE_NAME:?}" ]
-  then
-    CONFIG_SCRIPT=PullRequestLinuxGCC4.8.4TestingSettings.cmake
-  elif [ "Trilinos_pullrequest_gcc_4.9.3" == "${JOB_BASE_NAME:?}" ]
-  then
-    CONFIG_SCRIPT=PullRequestLinuxGCC4.9.3TestingSettings.cmake
-  fi
+    if [ "Trilinos_pullrequest_gcc_4.8.4" == "${JOB_BASE_NAME:?}" ]; then
+        CONFIG_SCRIPT=PullRequestLinuxGCC4.8.4TestingSettings.cmake
+    elif [ "Trilinos_pullrequest_gcc_4.9.3" == "${JOB_BASE_NAME:?}" ]; then
+        CONFIG_SCRIPT=PullRequestLinuxGCC4.9.3TestingSettings.cmake
+    # --- BEGIN SCAFFOLDING ---
+    # nurgleflurgle... name match!  can we change this to a regex or something to
```

Comment:
"nurgleflurgle"?

Reply:
hah... as noted in my 1st comment -- all the stuff in `scaffolding` blocks will be removed before merging.  This was just in there because the testing scripts do a name-match on the Jenkins job... once the framework team is ready to merge this in, I'll remove those comment blocks.